packages/core/documentation/src/helpers.ts

```diff
@@ -0,0 +1,14 @@
+export function cloneDeep(a: any) {
```

Comment:
suggestion: can you copy over the unit tests from `khcp-ui` for both of these?

Reply:
good suggestion, `isObjectEmpty` had a test so I brought it over